lib/spans/span-streamer.js

```diff
@@ -54,14 +54,9 @@ class SpanStreamer {
 
       // If the queue is full drop the span
       logger.infoOncePer(
-        // key for the OncePer
-        'SPAN_DROP_MSG',
-        // interval in ms
-        SPAN_DROP_INTERVAL * 1000,
-        // message
-        SPAN_DROP_MSG + ' Will not warn again for %s seconds.',
-        // variables to put in log message
-        SPAN_DROP_INTERVAL
+        'SPAN_DROP_MSG', // key for the OncePer
+        SPAN_DROP_MSG_INTERVAL_MS,
+        SPAN_DROP_MSG
```

Comment:
Some cleanup to reduce recalculating certain pieces each time.

Reply:
I like the cleaner comments and the message not being appended to